README.md

```diff
@@ -1,5 +1,7 @@
 # cisco-cookbook
 
+[![Build Status](https://travis-ci.org/cisco/cisco-network-chef-cookbook.svg?branch=develop)](https://travis-ci.org/cisco/cisco-network-chef-cookbook)
+
```

Comment:
Never mind - I had to touch README.md to resolve the merge conflict with your workflow additions, so I went ahead and added this too.